stabilizer/src/main/java/com/hazelcast/stabilizer/worker/tasks/SingleOperationAbstractWorkerTask.java

```diff
@@ -0,0 +1,33 @@
+package com.hazelcast.stabilizer.worker.tasks;
+
+/**
+ * Single operation version of {@link AbstractWorkerTask}.
+ * <p/>
+ * There is no {@link com.hazelcast.stabilizer.worker.selector.OperationSelector} used in this worker.
+ */
+public abstract class SingleOperationAbstractWorkerTask extends AbstractWorkerTask {
+
+    public SingleOperationAbstractWorkerTask() {
```

Comment:
Can we rename this class to MonotonicWorkerTask or something? Less painful than SingleOperationAbstractWorkerTask.